src/locale.en.js

```diff
@@ -51,6 +51,11 @@ export const campsite = {
     }
 };
 
+export const reducerAlerts = {
+    site_upload_success: 'Successfully uploaded your site. Thanks!',
+    site_upload_error: 'Sorry, there was an error uploading your site. Please try again later.'
+};
```

Comment:
I made these text entries to make a popup when you have success or failure... but I had to remove the alerts themselves. @amgoodman5 let's have a much larger UX conversation tomorrow about what happens to the user flow after an upload attempt.

Reply:
👍